<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_HvConfig</class>
 <widget class="QDialog" name="Dial_HvConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load High Voltage Orders</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_File">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_File">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>File Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sel_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Module 0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sel_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Module 1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Module 2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sel_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Module 3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sel_All">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>radioButton_Sel_0</zorder>
  <zorder>groupBox</zorder>
  <zorder>lineEdit_File</zorder>
  <zorder>toolButton_File</zorder>
  <zorder>pushButton_Do</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>textEdit_Msg</zorder>
  <zorder>radioButton_Sel_1</zorder>
  <zorder>radioButton_Sel_2</zorder>
  <zorder>radioButton_Sel_3</zorder>
  <zorder>radioButton_Sel_All</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
